microsoft-edge/webview2/how-to/webdriver.md

```diff
@@ -103,24 +103,20 @@ At this point you should have WebView2 Runtime installed, built a WebView2 proje
     | `is_legacy` | Set to `false`, which tells Selenium that you are driving the new Chromium-based Microsoft Edge browser. |  
     | `"webview2"` | A string that tells Selenium you are driving WebView2. |  
     
-1.  Next, set `edgeOptions.BinaryLocation` to the file path of your WebView2 project runtime, create a string named `msedgedriverDir` that provides the file path to where you installed [Microsoft Edge Driver][MicrosoftDeveloperMicrosoftEdgeWebDriverDownloads], and create a string named `msedgedriverExe` to store the name of the Microsoft Edge Driver runtime.  By default, the runtime is named `msedgedriver.exe`. Use these two strings to construct the `EdgeDriverService` object as shown below.  Finally, create the `EdgeDriver` object using `EdgeDriverService` and `EdgeOptions`.  
+1.  Next, configure `edgeOptions` to use Chromium and WebView by setting the member variables `UseChromium` and `UseWebView` to true. Then, set `edgeOptions.BinaryLocation` to the file path of your WebView2 project runtime. Finally, create the `EdgeDriver` object using `EdgeOptions`.  
```

Comment:
not sure if 'member variables' is the proper terminology

Reply:
Yup that looks right. To be more clear you could do `edgeOptions.UseChromium` and `edgeOptions.UseWebView`, but it's clear regardless, and the sample code also explains this.

Reply:
These are C# properties, not member variables. Another more language-neutral term would be "options".

Reply:
changed to 'options'